src/fsharp/FSharp.Core/Linq.fs

```diff
@@ -782,7 +782,7 @@ module LeafExpressionConverter =
     // provides no other way to evaluate the expression.
     //
     // REVIEW: It is possible it is just better to interpret the expression in many common cases, e.g. property-gets, values etc.
-    let EvaluateQuotation (e: Microsoft.FSharp.Quotations.Expr) = 
+    let EvaluateQuotation (e: Microsoft.FSharp.Quotations.Expr) : obj = 
```

Comment:
We are happy to accept sensible whitespace cleanup, but if this type annotation isn't needed, the open repo should remove it.